.github/workflows/test.yml

```diff
@@ -37,8 +36,9 @@ jobs:
           # pytest-qt CI dependencies: https://pytest-qt.readthedocs.io/en/latest/troubleshooting.html#github-actions
           sudo apt update
           sudo apt install -y \
-            xvfb \
-            libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils
+            xvfb libegl1 \
+            libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 x11-utils \
+            libxcb-cursor0
```

Comment:
Without `libxcb-cursor`, the `pytest` step in GitHub Actions would fail with `Aborted (core dumped)`.